net-misc/unison/unison-2.51.3_rc2.ebuild

```diff
@@ -64,7 +66,10 @@ src_install() {
 	for binname in unison unison-fsmonitor; do
 		newbin ${binname} ${binname}-${SLOT}
 	done
-
+	if use gtk; then
+		newicon -s scalable ../icons/U.svg ${PN}-${SLOT}.svg
+		make_desktop_entry ${PN}-${SLOT} ${PN}-${SLOT} ${PN}-${SLOT}
```

Comment:
Double name!
```
$ ls /var/tmp/portage/net-misc/unison-2.51.3_rc2/image/usr/share/applications/ -l
total 5
-rw-r--r-- 1 root root 173 Oct 19 02:34 unison-2.51-unison-2.51.desktop
```

Reply:
The eclass does it for us, it [seems](https://gitweb.gentoo.org/repo/gentoo.git/tree/eclass/desktop.eclass#n159)! Always test ;)

Reply:
> Double name!
> 
> ```
> $ ls /var/tmp/portage/net-misc/unison-2.51.3_rc2/image/usr/share/applications/ -l
> total 5
> -rw-r--r-- 1 root root 173 Oct 19 02:34 unison-2.51-unison-2.51.desktop
> ```

Fun with desktop.eclass:  The file seems to be a concatenation of <command>,  {PN}  and ".desktop".  And if the command is slotted the file name appears with double name.  
I have tried the proposal ( lowercase / uppercase ) from https://bugs.gentoo.org/360161#c8 but could not find a solution. 
For 2.51.3_rc2 your change works but it doesn't for 2.48.15_p4.  I kind of don't understand